lib/dl_connector_chyt/pyproject.toml

```diff
@@ -14,17 +14,18 @@ attrs = ">=22.2.0"
 marshmallow = ">=3.19.0"
 python = ">=3.10, <3.13"
 clickhouse-sqlalchemy = {path = "../clickhouse-sqlalchemy"}
-sqlalchemy = ">=1.4.46, <2.0"
-datalens-api-commons = {path = "../dl_api_commons"}
-datalens-utils = {path = "../dl_utils"}
-datalens-constants = {path = "../dl_constants"}
-datalens-i18n = {path = "../dl_i18n"}
-datalens-connector-clickhouse = {path = "../dl_connector_clickhouse"}
-datalens-sqlalchemy-chyt = {path = "../dl_sqlalchemy_chyt"}
-datalens-configs = {path = "../dl_configs"}
-datalens-api-connector = {path = "../dl_api_connector"}
-datalens-core = {path = "../dl_core"}
-datalens-db-testing = {path = "../dl_db_testing"}
+sqlalchemy = ">=1.4.52, <2.0"
```

Comment:
sqlalchemy was reverted back to 1.4.46 https://github.com/datalens-tech/datalens-backend/commit/84f269ae188edc531cb323831c38e34293bbfb33
probably needs a rebase